site/content/faq/how-can-i-set-variables-in-the-automation-framework.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "How Can I set Variables in the Automation Framework?"
+type: faq
+category: Automation
+weight: 8
+---
+
+The [Automation Framework](/docs/automate/automation-framework/) supports 
+[variables](/docs/desktop/addons/automation-framework/environment/#variables), which includes environmental variables.
+You can use these to set variables referenced in your plan from the command line, including secrets such as credentials.
```

Comment:
to reduce the occurrences of "variables" could we say "...to set values referenced....."?

Reply:
Done